internal/clauderig/commands/desktop_session.go

```
@@ -0,0 +1,257 @@
+// which window receives it.
+//
+// Observed, not theorised: asking for `brightshore` with `relatecpa` also open
+// imported the session into relatecpa — the sidecar landed under relatecpa's
+// accountUuid. That is a session crossing an ACCOUNT boundary, so the default
+// is to refuse rather than warn and hope. --anyway sends it regardless, for the
+// case where any window will do.
+func ambiguousRoutingError(target desktop.Profile, others []string) error {
+	// The first line is rendered as the headline — capitalised, and given a
+	// trailing period — so it must be one plain sentence that survives both, and
+	// must not start with a profile name.
+	return fmt.Errorf("another Desktop profile is open, so this session could be imported into the wrong account\n\n"+
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)